unittests/parser_test_case.py

```diff
@@ -84,3 +87,10 @@ def _test_calldef_exceptions(self, calldef, exceptions):
             (calldef.name,
              pprint.pformat([delc.name for delc in exception_decls]),
              pprint.pformat([delc.name for delc in exceptions_indeed])))
+
+if sys.version_info < (2, 7, 0) and not found_unittest2:
```

Comment:
Normally I'd agree with you, but in this case, `unittest2` isn't installed by default on Centos6.7. I'm trying to upgrade a project at work from gccxml to castxml, and I'd like to keep additional dependencies to a minimum. This also the reason I'm using Python2.6 instead of 2.7.

If more and more things show up that require `unittest2`, then absolutely, make that a requirement. But for this particular instance, with a workaround of 6 lines of code, I'd prefer to keep this.

But it's up to you, if you feel strongly that we should throw an exception (removing the `import try-catch`), I can change the code to do that.